pkg/helm/build.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 )
 
-const helmClientVersion = "v2.12.3"
+const helmClientVersion = "v2.13.0"
```

Comment:
Everytime we bump this, it immediately breaks the CI tests because the client and server versions are mismatched. What do you recommend we do to manage this? Otherwise I'm not keen on bumping too often without a plan.

Reply:
Ah, I can understand the hesitation :)

Indeed, our plan should include an easy mechanism to bump CI/related infra at the same time, or perhaps revisit the CI tests themselves such that this mixin's version of helm can be bumped to the latest without worry.  Let's think/chat about it and hold off on this change.